packages/cactus-core/src/main/typescript/web-services/swagger-endpoints.ts

```diff
@@ -0,0 +1,105 @@
+import { OpenApiJson } from "@hyperledger/cactus-core-api";
+import express from "express";
+const app = express();
+import swaggerUi from "swagger-ui-express";
+
+const options = {
```

Comment:
@charellesandig We cannot hardcode the URLs/names, they must come from the  web service plugins dynamically at runtime. Please take a look at the `PluginRegistry` class and how it is used by the `ApiServer` to do such enumerations and then work your way from there. I'm happy to assist with more code references.